src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,15 @@
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class name `ReturnOperation` is not very descriptive. Consider renaming it to better reflect its purpose, such as `ReturnOperationHandler`.